.github/workflows/ui-beta.yml

```
@@ -23,15 +23,16 @@ jobs:
       - uses: ./.github/actions/frontend-lint
         with:
           frontend_directory: ${{ env.directory }}
-          installation_directory: ${{ env.centreon_directory }}
           module_name: centreon-ui
+          dependencies_lock_file: centreon/pnpm-lock.yaml
```

```suggestion
          dependencies_lock_file: ${{ env.centreon_directory }}/pnpm-lock.yaml
```
just a suggestion ;)